packages/sui-segment-wrapper/src/repositories/googleRepository.js

```diff
@@ -52,3 +52,13 @@ const getGoogleField = async field => {
 
 export const getGoogleClientID = () => getGoogleField(FIELDS.clientId)
 export const getGoogleSessionID = () => getGoogleField(FIELDS.sessionId)
+
+export const sendGoogleUserId = userId => {
+  const googleAnalyticsMeasurementId = getConfig('googleAnalyticsMeasurementId')
+
+  if (!googleAnalyticsMeasurementId || !userId) return
+
+  window.gtag?.('set', googleAnalyticsMeasurementId, {
+    user_id: userId
+  })
+}
```

Comment:
looking at this I'd say it's just setting the `userId`, not sending it... am I right?

```suggestion
export const setGoogleUserId = userId => {
  const googleAnalyticsMeasurementId = getConfig('googleAnalyticsMeasurementId')

  if (!googleAnalyticsMeasurementId || !userId) return

  window.gtag?.('set', googleAnalyticsMeasurementId, {
    user_id: userId
  })
}
```

Reply:
@jordevo Yep, in fact it's incorrect, I've changed the implementation (and also the function name) following the official docs: https://developers.google.com/analytics/devguides/collection/ga4/reference/config#user_id_ex